dsc/src/main.rs

```diff
@@ -484,13 +488,22 @@ impl DscInfo {
         ds_id: usize,
         extent_size: u64,
         extent_count: u32,
-        block_size: u32,
+        block_size: u64,
         quiet: bool,
         encrypted: bool,
     ) -> Result<f32> {
         // Create the path for this region by combining the region
         // directory and the port this downstairs will use.
         let mut rs = self.rs.lock().await;
+
+        // If we don't have region info yet, set it now.
```

Comment:
Should we check against existing `region_info` and bail if they don't match?

Reply:
I considered that, but decided not to, as we could be testing that a mismatch is caught by the upstairs.  I could make a check and warning here though.

Reply:
Also, currently DSC does not allow you to create regions that differ from each other, so to get something like this to exist, you would have to run dsc multiple times, then try to stitch all the parts together after the fact to get a single DSC to serve the results.

Reply:
Added warnings in 8acd4d6